draft-ietf-oauth-transaction-tokens.md

```diff
@@ -347,7 +347,7 @@ The figure below {{figleaftxtokenbody}} shows a non-normative example of the JWT
 ~~~ json
 {
     "iat": "1686536226000",
-    "aud": "trust-domain.example",
+    "aud": "https://trust-domain.example",
```

Comment:
Do we need to have "https" or any protocol in a StringOrUri? It should be OK to not have a protocol prefix.

Reply:
True, if StringOrUri type should be defined, than leading https:// is not necessary. In this case I would align the other instances where this audience is used.

As you want to discuss it in the call today, I wait with the update of the PR until after the discussion.

(P.S: As much as I would like to join the regular call, the time slot is quite difficult for me due to other conflicting obligations.)